[ayucloud.services]
auth_path  user
auto_invite  F
banned_domains  zz.vlvitp.shop  lista.cc  965.tmmad.com  yzm.de  smartnator.com
buy  pass
email  emcbuptti813@guerrillamail.biz
last_checkin  2025-04-15 07:34:46.423725+08:00
name  AyucLouD-Services
node_n  99
sub_info  0B  0B  2025-04-16 07:34:46+08:00  (剩余 0B -1 day, 18:45:14.153492)
sub_url  https://network.ayucloud.nl/link/rS8bySjCT35iyMP5?clash=1
time  2025-04-15 07:34:47.391876+08:00
type  sspanel
保存base64/clash订阅失败  'NoneType' object is not iterable
尝试签到失败  HTTPSConnectionPool(host='ayucloud.services', port=443): Max retries exceeded with url: /user/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f05769cebf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
更新订阅链接/续费续签失败  注册失败(oqhbolfws75@gmail.com): HTTPSConnectionPool(host='ayucloud.services', port=443): Max retries exceeded with url: /user/register (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0576a672e0>, 'Connection to ayucloud.services timed out. (connect timeout=30)'))

[qlgq.top]

[cyanmori.com]

[vipxgyun.com]
